### DISCUSSION TITLE
Request to add spanwise loading and chord wise loading output based on ID

### DISCUSSION THREAD

**Discussion by Preethamkadambi** [DLR-AE/PanelAero#28] [Questions & Answers]
As the title suggests,
1. Is there a way to export spanwise aero-loading and chord wise loading? 
I wanted to compare Schrenks method, ANC-1 and ANC-2 results with DVLM and VLM results. 
2.Is it possible to define control surfaces and their deflections?

**Comment by ArneVoss** [comment]
Hello Preetham, 
both the VLM and the DLM produce a so-called AIC matrix, which can be used to calculate the pressure distribution cp based on the downwash wj per panel: cp = AIC * wj
The rest is up to you - if you are interested in a span- or chord-wise pressure distribution, I propose to integrate over the panels in the desired direction. Control surface deflections are modeled by an additional downwash on the corresponding aerodynamic panels.
Does this help you?
